src/System.Globalization.Calendars/tests/Misc/Calendars.netstandard1.7.cs

```diff
@@ -0,0 +1,93 @@
+{
+
+    public static class CalendarsMoreTests
+    {
+        public static IEnumerable<object[]> Calendars_TestData()
+        {
+            //                              Calendar               yearHasLeapMonth        CalendarAlgorithmType 
+            yield return new object[] { new ChineseLunisolarCalendar()  , 0     , CalendarAlgorithmType.LunisolarCalendar   };
+            yield return new object[] { new GregorianCalendar()         , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new HebrewCalendar()            , 5345  , CalendarAlgorithmType.LunisolarCalendar   };
+            yield return new object[] { new HijriCalendar()             , 0     , CalendarAlgorithmType.LunarCalendar       };
+            yield return new object[] { new JapaneseCalendar()          , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new JapaneseLunisolarCalendar() , 0     , CalendarAlgorithmType.LunisolarCalendar   };
+            yield return new object[] { new JulianCalendar()            , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new KoreanCalendar()            , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new KoreanLunisolarCalendar()   , 0     , CalendarAlgorithmType.LunisolarCalendar   };
+            yield return new object[] { new PersianCalendar()           , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new TaiwanCalendar()            , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new TaiwanLunisolarCalendar()   , 0     , CalendarAlgorithmType.LunisolarCalendar   };
+            yield return new object[] { new ThaiBuddhistCalendar()      , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new UmAlQuraCalendar()          , 0     , CalendarAlgorithmType.LunarCalendar       };
+        }
+
+        [Theory]
+        [MemberData(nameof(Calendars_TestData))]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
+        public static void CloningTest(Calendar calendar, int yearHasLeapMonth, CalendarAlgorithmType algorithmType)
+        {
+            Calendar cloned = (Calendar) calendar.Clone();
+            Assert.Equal(cloned.GetType(), calendar.GetType());
+        }
+
+        [Theory]
+        [MemberData(nameof(Calendars_TestData))]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
+        public static void GetLeapMonthTest(Calendar calendar, int yearHasLeapMonth, CalendarAlgorithmType algorithmType)
+        {
+            if (yearHasLeapMonth > 0)
+            {
+                Assert.NotEqual(calendar.GetLeapMonth(yearHasLeapMonth),  0);
+                Assert.Equal(calendar.GetLeapMonth(yearHasLeapMonth - 1), 0);
```

Comment:
Same expected/actual comment as above.  I'll stop commenting on this, but it applies for all such asserts.